Doc/library/ctypes.rst

```diff
@@ -1394,8 +1394,8 @@ copy of the windows error code.
 
 The *winmode* parameter is used on Windows to specify how the library is loaded
 (since *mode* is ignored). It takes any value that is valid for the Win32 API
-``LoadLibraryEx`` flags parameter. When omitted, the default is to use the flags
-that result in the most secure DLL load to avoiding issues such as DLL
+``LoadLibraryEx`` flags parameter. When it is ``None``, the default is to use
+the flags that result in the most secure DLL load to avoiding issues such as DLL
```

Comment:
While you're here, replace "to avoiding" with "in order to avoid".

Reply:
now done